client/signup/steps/user/index.jsx

```diff
@@ -496,15 +505,53 @@ export class UserStep extends Component {
 		}
 
 		return (
-			<StepWrapper
-				flowName={ this.props.flowName }
-				stepName={ this.props.stepName }
-				headerText={ this.getHeaderText() }
-				subHeaderText={ this.getSubHeaderText() }
-				positionInFlow={ this.props.positionInFlow }
-				fallbackHeaderText={ this.props.translate( 'Create your account.' ) }
-				stepContent={ this.renderSignupForm() }
-			/>
+			<ProvideExperimentData
+				name="registration_social_login_first_on_mobile_v2"
+				options={ {
+					isEligible: true || ( isMobile() && this.props.flowName === 'onboarding' ),
+				} }
+			>
+				{ ( isLoading, experimentAssignment ) => {
+					if ( isLoading ) {
+						return null;
+					}
+
+					if ( experimentAssignment?.variationName === 'treatment' ) {
+						const loginUrl = this.getLoginUrl( this.props );
+						const subHeading = this.props.translate( 'Already have an account? {{a}}Log in{{/a}}', {
+							components: { a: <a href={ loginUrl } rel="noopener noreferrer" /> },
+						} );
+						return (
+							<div className="user__simpler-mobile-form">
+								<StepWrapper
+									flowName={ this.props.flowName }
+									stepName={ this.props.stepName }
+									headerText={
+										this.isEmailForm()
+											? this.props.translate( 'Create Account' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Create account.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/user-registration-mobile-social-links)